### DISCUSSION TITLE
How can i make these customizations (if possible)?

### DISCUSSION THREAD

**Discussion by bheeshmpita** [tomasklaen/uosc#859] [Q&A] [answered]
1. when next chapter begins, the chapter name is mentioned in top left corner for x seconds
2. the title of video mentions the date when it was uploaded (while streaming videos).
Thanks

**Comment by tomasklaen** [comment, marked as answer]
1. Currently not possible.
2. We don't have that data to display it.

**Reply by christoph-heinrich** [reply]
FWIW adding the upload date to the metadata in ytdl_hook.lua would be pretty easy.
